src/features/winterStorageAreaList/winterStorageAreaDetails/WinterStorageAreaDetails.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { WinterStorageAreaData } from '../types';
+import Grid from '../../../common/grid/Grid';
+import styles from '../../harborList/harborDetails/harborDetails.module.scss';
+import Section from '../../../common/section/Section';
+import ExternalLink from '../../../common/externalLink/ExternalLink';
+import MapLinks from '../../../common/mapLinks/MapLinks';
+import Text from '../../../common/text/Text';
+import { formatDimension } from '../../../common/utils/format';
+import placeholderImage from '../../../common/placeholderImage.svg';
+
+export type WinterStorageAreaDetailsProps = {
+  imageFile: WinterStorageAreaData['imageFile'];
+  maps: WinterStorageAreaData['maps'];
+  maxWidth: WinterStorageAreaData['maxWidth'];
+  municipality: WinterStorageAreaData['municipality'];
+  servicemapId: WinterStorageAreaData['servicemapId'];
+  streetAddress: WinterStorageAreaData['streetAddress'];
+  zipCode: WinterStorageAreaData['zipCode'];
+};
+
+const WinterStorageAreaDetails = ({
+  maps,
+  maxWidth,
+  municipality,
+  servicemapId,
+  streetAddress,
+  zipCode,
+  imageFile,
+}: WinterStorageAreaDetailsProps) => {
+  const { t, i18n } = useTranslation();
+  const address = `${streetAddress}, ${zipCode} ${municipality}`;
+  const imageSrc = imageFile ? imageFile : placeholderImage;
+  const serviceMapUrl = `${process.env.REACT_APP_SERVICE_MAP_URI}${servicemapId}`;
+
+  return (
+    <Grid colsCount={4}>
+      <div className={styles.column}>
+        <img className={styles.image} src={imageSrc} alt="map" />
```

Comment:
I think it's better to add `alt` value to the translation files.

Reply:
Great idea, I did this for all instances of `alt` in the codebase (3 in total)